pyproject.toml

```diff
@@ -10,7 +10,7 @@ description = "ValidMind Developer Framework"
 license = "Commercial License"
 name = "validmind"
 readme = "README.pypi.md"
-version = "2.4.0"
+version = "2.4.1"
```

Comment:
Can you bump to `2.4.2`? I have a prod hotfix that I will bring to `main` soon.

Reply:
yep :)